### PR TITLE
expose interface manager

### PR DESCRIPTION
**Pull request by MusketPosition**
- Refactor manager
- Move event to interface
- All tests are ok

### CODE REVIEW

File: contracts/protocol/libraries/types/PositionManagerStorage.sol [hunk 1]
```diff
@@ -41,6 +39,11 @@ contract PositionManagerStorage {
         uint8 isFullBuy;
     }
 
+    struct LiquidityOfEachPip {
```
Comment (JustInPosition) [inline contracts/protocol/libraries/types/PositionManagerStorage.sol]:
```suggestion
    struct PipLiquidity {
```

**Comment by JustInPosition** [comment]
@MusketPosition please resolve conflicts

**Comment by JustInPosition** [comment]
@MusketPosition please resolve conflicts

**Comment by MusketPosition** [comment]
> @MusketPosition please resolve conflicts

already done

**Pull request merged**